client/src/LayoutComponents/pageNotFound/index.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import image from '../../images/error.png';
+import style from './index.css';
+
+const getErrorImage = () => {
+  return <img src={image} />;
+};
+export default getErrorImage;
```

Comment:
@Hussein-shahatet97 Just to be consistent with react component naming conventions, `ErrorImage` might be a better name (capitalized, and removed the "get" since this function doesn't really get anything)

Reply:
i will do it after fix conflict.

Reply:
i will do it after fix conflict.